samples/notebooks/python/09-groundedness-checking.ipynb

```diff
@@ -0,0 +1,351 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f5c76c5f",
+   "metadata": {},
+   "source": [
+    "# Groundedness Checking Skills\n",
+    "\n",
```

Comment:
Can you write a bit more inside the notebook about what the concept of groundedness is and why it's important? I don't think everyone will know from the get-go about the subject.

Reply:
Good point - done.